app/components/home-page/organization-dashboard/side-nav/index.hbs

```diff
@@ -1,85 +1,145 @@
-<aside local-class='side-menu-root'>
-  <div local-class='app-logo'>
-    <ImgLogo />
+<aside
+  local-class='side-menu-root {{if @isCollapsed "expanded" "collapsed"}}'
+  ...attributes
+>
+  <div local-class='{{if @isCollapsed "app-logo" "favicon-logo"}}'>
+    <LinkTo @route='authenticated.index'>
+      <img
+        data-test-img-logo
+        src={{if @isCollapsed this.whitelabel.logo this.whitelabel.favicon}}
```

Comment:
this should be `{{if @isCollapsed this.whitelabel.favicon this.whitelabel.logo}}`